### PR TITLE
Add copyparams method to Sequential

### LINKED ISSUE #5824: `Link.copyparams` does not work with `Sequential` with non-links
* Conditions
```
Platform: Darwin-18.2.0-x86_64-i386-64bit
Chainer: 6.0.0b1
NumPy: 1.15.4
CuPy: Not Available
iDeep: Not Available
```

* Code to reproduce
```python
l1 = chainer.Sequential(chainer.links.Linear(3, 4), chainer.functions.relu, chainer.links.Linear(4, 5))
l2 = chainer.Sequential(chainer.links.Linear(3, 4), chainer.functions.relu, chainer.links.Linear(4, 5))
l1.copyparams(l2)
```

* Error messages, stack traces, or logs
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/tos/GitHub/chainer/chainer/link.py", line 1227, in copyparams
    child.copyparams(link[idx], copy_persistent)
  File "/Users/tos/GitHub/chainer/chainer/link.py", line 611, in copyparams
    dst[name].copydata(src[name])
KeyError: 'b'
```

### PR DESCRIPTION
**Pull request by mitmul**
This PR fixes #5824

### CODE REVIEW

File: chainer/sequential.py [hunk 1]
```diff
@@ -381,6 +381,11 @@ def copy(self, mode='share'):
                 ret.append(copy.copy(layer))
         return ret
 
+    def copyparams(self, sequential, copy_persistent=True):
```
Comment (niboshi) [inline chainer/sequential.py]: >`sequential`

How about not changing the parameter name from the base method? I don't think it's a good practice.
By the way, can we assume the source link is of `Sequential`? (There's no such mention of the assumption in the documentation).

Comment (mitmul) [reply chainer/sequential.py]: OK, I’ll fix the name of this argument.

> can we assume the source link is of Sequential?
Ah, I guess it’s better to raise an error when the given source link is not a Sequential object?
And at least I need to add a description about it to the documentation.

**Comment by mitmul** [comment]
@niboshi Hi, I fixed the argument name and added a check for the object type of source link. Could you review this again?

**Comment by takagi** [comment]
Jenkins, test this please.

File: chainer/sequential.py (hunk 1 above)
Comment (niboshi) [reply chainer/sequential.py]: Thanks.

**Comment by mitmul** [comment]
Jenkins, test this please.

**Comment by mitmul** [comment]
@takagi could you merge this PR? All tests have succeeded.

File: chainer/sequential.py [hunk 2]
```diff
@@ -2,11 +2,11 @@
 import functools
 import inspect
 
+import chainer
 from chainer import function
-from chainer import link
 
 
-class Sequential(link.ChainList):
+class Sequential(chainer.ChainList):
```
Comment (toslunar) [inline chainer/sequential.py]: This change relies on that `chainer.sequential` is imported later than `chainer.link.ChainList` in `chainer/__init__.py`.

Comment (mitmul) [reply chainer/sequential.py]: Thanks, I updated the code by replacing `chainer.*` with `chainer.link.*`. Is it what you meant to fix?

**Comment by mitmul** [comment]
Jenkins, test this please

File: chainer/sequential.py (hunk 2 above)
Comment (niboshi) [reply chainer/sequential.py]: I don't think it fixes.
`chainer.link` might not be imported.

**Comment by mitmul** [comment]
Eh, jenkins, test this please

File: chainer/sequential.py (hunk 2 above)
Comment (mitmul) [reply chainer/sequential.py]: @toslunar @niboshi Could you check this again?

**Comment by takagi** [comment]
I now merge this!

**Pull request merged**